.vale.ini

```diff
@@ -0,0 +1,70 @@
+StylesPath = ".github/vale/styles"
+Vocab = "OpenSearch"
+MinAlertLevel = warning
+SkippedScopes = code, style
```

Comment:
SkippedScopes, BlockIgnores, and TokenIgnores are not applicable here so I would remove these.